diff --git a/node_modules/react-calendar/dist/Calendar.css b/node_modules/react-calendar/dist/Calendar.css
index 1c66761..8f17e86 100644
--- a/node_modules/react-calendar/dist/Calendar.css
+++ b/node_modules/react-calendar/dist/Calendar.css
@@ -1,8 +1,9 @@
 .react-calendar {
-  width: 350px;
+  width: 830px;
+  height: 540px;
   max-width: 100%;
-  background: white;
-  border: 1px solid #a0a096;
+  background: red;
+  border: 5px solid #a0a096;
   font-family: Arial, Helvetica, sans-serif;
   line-height: 1.125em;
 }
diff --git a/node_modules/react-calendar/src/MonthView.jsx b/node_modules/react-calendar/src/MonthView.jsx
index 6340c2c..0c00055 100644
--- a/node_modules/react-calendar/src/MonthView.jsx
+++ b/node_modules/react-calendar/src/MonthView.jsx
@@ -66,6 +66,7 @@ export default function MonthView(props) {
         style={{
           display: 'flex',
           alignItems: 'flex-end',
+          height: '100%',
         }}
       >
         {renderWeekNumbers()}
@@ -73,6 +74,7 @@ export default function MonthView(props) {
           style={{
             flexGrow: 1,
             width: '100%',
+            height: '100%',
           }}
         >
           {renderWeekdays()}
